features/Journalist_can_update_article.feature

```diff
@@ -8,10 +8,12 @@ Feature: Journalist can create article
         Given the following user exists
             | name | email          | password | password_confirmation | role       |
             | Bill | bill@email.com | password | password              | journalist |
+
```

Comment:
Dont clutter up the PR with changes that has nothing to do with the features

Reply:
I asked them to add a new line between steps in the background 😉

Reply:
This was a request from Thomas